staging/src/k8s.io/component-base/logs/api/v1/types.go

```diff
@@ -58,6 +61,23 @@ type LoggingConfiguration struct {
 	Options FormatOptions `json:"options,omitempty"`
 }
 
+type TimeOrMetaDuration metav1.Duration
```

Comment:
might be worth a comment that this is present only for backwards compatibility for the flushFrequency field, and new fields should use metav1.Duration

Reply:
Done.